DEVELOPMENT.rst

```diff
@@ -268,6 +268,7 @@ This is an example ``nginx.conf`` configuration for Daphne listing on port
                 proxy_pass http://localhost:8000;
             }
             location /rest {
+                proxy_set_header Host $http_host; 
```

Comment:
Why only for the /rest location?

Reply:
You would probably need it for all locations, but the /rest did the password reset mail, where the link and hostname were incorrect.

`
location / {
            try_files $uri $uri/ /index.html;
        }
        location /apps {
            proxy_set_header Host $http_host; 
            proxy_pass http://localhost:8000;
        }
        location /media {
            proxy_set_header Host $http_host; 
            proxy_pass http://localhost:8000;
        }
        location /rest {
            proxy_set_header Host $http_host; 
            proxy_pass http://localhost:8000;
        }
        location /ws {
            proxy_pass http://localhost:8000;
            proxy_http_version 1.1;
            proxy_set_header Upgrade $http_upgrade;
            proxy_set_header Connection "Upgrade";
        }
`